pkg/component/extensions/operatingsystemconfig/original/components/kubelet/config_test.go

```diff
@@ -282,6 +283,7 @@ var _ = Describe("Config", func() {
 				cfg.VolumePluginDir = "/var/lib/kubelet/volumeplugins"
 				cfg.ProtectKernelDefaults = true
 				cfg.StreamingConnectionIdleTimeout = metav1.Duration{Duration: time.Minute * 5}
+				cfg.StaticPodPath = ""
```

Comment:
We can instead `StaticPodPath:  ""` on line 153

Reply:
Good point :) Thanks!